wheatley/parsing.py

```diff
@@ -221,9 +227,12 @@ def json_to_call(name):
         return row_gen
 
     raise_error('type', f"{json['type']} is not one of 'method' or 'composition'")
+    # This code is unreachable, but used to demonstrate to the type-checker that `raise_error` will
+    # always throw an Exception
+    assert False
```

Comment:
You hopefully shouldn't need this with `NoReturn`

Reply:
Ayy yes we can just get rid of this now!